packages/kbn-apm-synthtrace/src/scenarios/new_icons.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { apm, timerange } from '../..';
+import { ApmFields } from '../lib/apm/apm_fields';
+import { Instance } from '../lib/apm/instance';
+import { Scenario } from '../cli/scenario';
+import { getLogger } from '../cli/utils/get_common_services';
+import { RunOptions } from '../cli/utils/parse_run_cli_flags';
+import { getSynthtraceEnvironment } from '../lib/utils/get_synthtrace_environment';
+
+const ENVIRONMENT = getSynthtraceEnvironment(__filename);
```

Comment:
Renaming the scenario to something like cloud services might be more meaningful.

Reply:
Thank you, renamed the scenario to something clearer and more meaningful. `cloud_services_icons`.